Realm/ObjectServerTests/SwiftSyncTestCase.swift

```diff
@@ -90,9 +117,9 @@ class SwiftSyncTestCase: RLMSyncTestCase {
         return theUser
     }
 
-    func synchronouslyLogOutUser(_ user: User,
-                                 file: StaticString = #file,
-                                 line: UInt = #line) throws {
+    public func synchronouslyLogOutUser(_ user: User,
```

Comment:
Should the 'synchronously' prefix be removed to make it consistent with `openRealm`?